doc/examples/policy.cfg-rolebased

```diff
@@ -26,6 +26,9 @@ role-rgw/cluster/rgw*.sls
 # openATTIC
 role-openattic/cluster/openattic*.sls
 
+#ceph-mgr
+role-mgr/cluster/*.sls
```

Comment:
all nodes should become a manage? Maybe better add all MONs as mgr nodes? this would also reflect the default deployment of ceph.